karapace/schema_registry_apis.py

```diff
@@ -241,8 +227,27 @@ def send_kafka_message(self, key, value):
         future = self.producer.send(self.config["topic_name"], key=key, value=value)
         self.producer.flush(timeout=self.kafka_timeout)
         msg = future.get(self.kafka_timeout)
-        self.log.debug("Sent kafka msg key: %r, value: %r, offset: %r", key, value, msg.offset)
-        self.get_offset_from_queue(msg.offset)
+        sent_offset = msg.offset
+
+        self.log.info(
+            "Waiting for schema reader to caugth up. key: %r, value: %r, offset: %r",
+            key,
+            value,
+            sent_offset,
+        )
+
+        if self.ksr.offset_watcher.wait_for_offset(sent_offset, timeout=1 * MINUTE) is True:
```

Comment:
One minute timeout is plenty, could this be lower? Some HTTP clients have shorter default timeouts.

Reply:
Also this is wrong as it is not one minute but actually thousand minutes, see karapace constants.py and my comment about `wait_for_offset` type signature.

Reply:
What would be the benefit of having a shorter timeout here? I'm trying to avoid causing further problems, so I was conservative.

Thanks @tvainika , you're right, MINUTE is in ms and this API is in secs